php-81/Dockerfile

```diff
@@ -473,12 +230,12 @@ RUN php /bref/lib-copy/copy-dependencies.php /bref-layer/bref/extensions/pdo_mys
 RUN php /bref/lib-copy/copy-dependencies.php /bref-layer/bref/extensions/pdo_pgsql.so /bref-layer/lib
 
 # Copy the OpenSSL certificates file
-RUN cp ${CA_BUNDLE} /bref-layer/bref/ssl/cert.pem
+RUN cp /etc/ssl/cert.pem /bref-layer/bref/ssl/cert.pem
 
 
 # ---------------------------------------------------------------
 # Start from a clean image to copy only the files we need
-FROM public.ecr.aws/lambda/provided:al2-${IMAGE_VERSION_SUFFIX} as isolation
+FROM public.ecr.aws/amazonlinux/amazonlinux:2023 as isolation
```

Comment:
If you're looking at constructing an image closer to what the Lambda image is, you can build from the `minimal` container image (see https://docs.aws.amazon.com/linux/al2023/ug/minimal-container.html )

Reply:
Thanks @stewartsmith 
I think this line is a mistake from me, I think using this is what we want:

```
FROM public.ecr.aws/lambda/provided:al2023-preview-${IMAGE_VERSION_SUFFIX}
```

(in a previous version of this PR, before there were Lambda preview images for 2023, I used to use the official AL2023 image -> I think this is a leftover I forgot to update)